daemon/mgr/container.go

```diff
@@ -650,6 +650,15 @@ func (mgr *ContainerManager) prepareContainerNetwork(ctx context.Context, c *Con
 		}
 	}
 
+	sb, err := mgr.NetworkMgr.Controller().SandboxByID(c.NetworkSettings.SandboxID)
+	if err != nil {
+		//sandbox not found, maybe caused by disconnect network or no endpoint
+		logrus.Warnf("failed to get sandbox by id(%s)", c.NetworkSettings.SandboxID)
```

Comment:
I think we should print error information.

Reply:
There's a scenario in pouch, the container disconnect the network and remove the endpoint, and if there's no endpoint in sandbox, the sandbox will be deleted. So in this scenario, no sandbox in container is not an error?

Reply:
I mean we should print `err`, like this: 
```
logrus.Warnf("failed to get sandbox by id(%s), err(%v)", c.NetworkSettings.SandboxID, err)
```

Reply:
get it, thanks.